### DISCUSSION TITLE
Worse results with new Grey-Mode

### DISCUSSION THREAD

**Discussion by pos-ei-don** [jomjol/AI-on-the-edge-device#2929] [General]
### The Problem

Its more information than a bug report! See the attached Screenshot!

On the right you see the old color-Mode with very good results, on the left you see the new grey-mode.
The analog-values are way off.....

### Version

15.7.0

### Logfile

```shell
-
```

### Screenshots

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1822533/5561014c-380d-4585-897c-6fb3b232efa7)

**Comment by nliaudat** [comment]
Hi, 
The AI model is better for RGB recognition. 
That was discussed before but sorry, I can't find the reference anymore

**Comment by caco3** [comment]
Depending on our meter, grayscale or inverted images might give better results as color images. but this is very individual. Just try what works best for you.

**Comment by pos-ei-don** [comment]
I just return to color for me.
